docs/diagnostics/PX1019.md

```diff
@@ -0,0 +1,31 @@
+# PX1019
+This document describes the PX1019 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                                                | Type  | Code Fix    | 
+| ------ | ---------------------------------------------------------------------------------------------------------------- | ----- | ----------- | 
+| PX1019 | A DAC property field with the `AutoNumber` attribute must have the string type.                                  | Error | Available   |
+
+## Diagnostic Description
+You can use the `AutoNumber` attribute on a DAC field to enable auto-numbering for the field. The field must be of the string type.
+
+To fix the issue, you can do one of the following:
+ - Change the field type to string.
+ - Use the `AutoNumber` attribute on a different field that has a string type.
```

Comment:
The description of code fix is missing. 
I suggest to describe it like in other diagnostics:
https://github.com/Acumatica/Acuminator/blob/7f20625dad642154001d19aa998a731cd1ee7964/docs/diagnostics/PX1011.md